pkg/ddevapp/ddevapp.go

```diff
@@ -1028,6 +1028,10 @@ func (app *DdevApp) Start() error {
 
 	app.DockerEnv()
 	dockerutil.EnsureDdevNetwork()
+	// "docker-compose up", which is called in this function later, may create
+	// duplicate project networks, we can create this network in advance
+	// see https://github.com/ddev/ddev/pull/5533
+	dockerutil.EnsureProjectNetwork()
```

Comment:
This PR is easy to revert, you only need to remove this function call and the function itself.